e2e/fixtures/ssg-wildcard/src/pages/[...wildcard].tsx

```diff
@@ -0,0 +1,14 @@
+const Page = ({ wildcard }: { wildcard: string[] }) => (
+  <div>
+    <h1>/{wildcard.join('/')}</h1>
+  </div>
+);
+
+export const getConfig = async () => {
+  return {
+    render: 'static',
+    staticPaths: [[], 'foo', ['bar', 'baz']],
+  };
+};
```

Comment:
This isn't type checked.
I wonder whether we should make `ssg-wildcard` with `createPages` API instead of fs-router.

We can say that types are covered with unit tests, but still fixtures should be as basic as possible?

Reply:
Actually, it is not type-covered in `createPages` either:

https://github.com/dai-shi/waku/blob/10c2865d7859a38df09d9c65663abb3318283649/packages/waku/tests/create-pages.test.ts#L60-L67

Reply:
Oh, I see. I hope we can fix it.

My question still holds: Should we make fixtures use low-level createPages instead of fs-router? (Unless we specifically tests fs-router.)
What do you think?

Reply:
Good question. My gut feeling would be to use integration tests to cover as much ground as possible, but in this case `createPages` is a common entry point (and even my preferred one). Also `fs-router.ts` is a good target for a unit test.
But this then should also apply to `partial-build` and `ssg-performance`. And most other tests don't use `createPages` but `defineRouter`. Maybe another PR where we look at the e2e setup holistically, including the typing hiccups that we encountered?

Reply:
Sure. Sounds all good!